### ISSUE TITLE
undefined method `day_names' for RailsAdmin::Config::Fields::Types::Datetime:Class

### ISSUE THREAD

**Issue by richardvenneman** [colavitam/rails_admin_grid#2]
Hi there,

Thanks for the plugin. Unfortunately, it seems it doesn't currently work against master rails_admin:

```
Showing /Users/richard/.rbenv/versions/2.2.2/lib/ruby/gems/2.2.0/gems/rails_admin_grid-0.0.1/app/views/rails_admin/main/grid.html.haml where line #79 raised:

undefined method `day_names' for RailsAdmin::Config::Fields::Types::Datetime:Class
```

It looks like the public day_names method was removed in this commit: https://github.com/sferik/rails_admin/commit/72c203394096221ed97846ba9e8af529d4cdccfa

**Comment by fmh** [comment]
+1 , just with RA 0.7 (last release)

**Comment by colavitam** [comment]
@fmh, I've been unable to reproduce this on rails_admin 0.7. The breaking changes that @richardvenneman posted occurred after version 0.7 was released.

There have been changes in how collection pages are handled (specifically with filters and subtleties like day_names), and I think it would make sense to wait until an official release appears with these changes. If we switched to the new format now, we'd be breaking the app for everyone not pulling from the github repo. Once the release comes along, we can adjust the collection page and then bump the version requirement.
